src/bundle/Resources/config/routing.yml

```
@@ -570,6 +570,13 @@ ezplatform.content.translate:
         _controller: 'EzPlatformAdminUiBundle:ContentEdit:translate'
         fromLanguageCode: ~
 
+ezplatform.content.translate_with_location:
```

```suggestion
ibexa.content.translate_with_location:
```

Shouldn't we also deprecate old one?

---

> shouldn't we still use `ezplatform` in `v2.5`?

@barw4 If you did that, you would need to provide BC layer for that in 4.0.

---

I thought we could just change it to `ibexa` during the merge up, nevertheless, I'll change it.

---

> I thought we could just change it to `ibexa` during the merge up, nevertheless, I'll change it.

Imagine the following scenario: A developer relies on that route in the next 2.5.x release. Then a developer  needs to upgrade to 3.3 and then 4.0. We've committed to promise BC wherever it's possible[1]. However after an upgrade a developer needs to change his code because now his app is crashing due to missing route.

[1] there will be exceptions to that rule due technical reasons, but the rule is actually "wherever possible without big costs" and moreover we agreed that we give BC on routes.